shell/apps/web/index.debug.js

```diff
@@ -0,0 +1,10 @@
+// global configuration
+import './config.js';
+// firebase runtime (customized by sjmiles@ for import-ability)
```

Comment:
I wish we had a diff, however tiny it might be, to show what you had to change to allow this. Can you paste a manual diff into a gist and link it here, maybe? So that if we have to repeat in future we have something we can track down.

I guess we can diff the old version vs yours but having the gist would save effort from multiple folks in future.

Reply:
I'll add a readme file that shows the two changes.